src/libs/EmojiUtils.js

```diff
@@ -254,11 +235,11 @@ function suggestEmojis(text, limit = 5) {
 }
 
 export {
-    isSingleEmoji,
     getDynamicHeaderIndices,
     mergeEmojisWithFrequentlyUsedEmojis,
     addToFrequentlyUsedEmojis,
     containsOnlyEmojis,
     replaceEmojis,
     suggestEmojis,
+    trimEmojiUnicode,
```

Comment:
This wasn't exported earlier and now we're exporting? Can't find its usage in the PR.

Reply:
I was curious about this too

Reply:
This isn't used anymore in the code, but I used it in the script tagged in `emojis.js`. So kept it as a reference.